packages/app/src/components/techdocs/TechDocsPage.tsx

```diff
@@ -14,21 +14,21 @@
  * limitations under the License.
  */
 
+import React from 'react';
+import { Page } from '@backstage/core-components';
 import {
-  TechDocsReaderPage,
   TechDocsReaderPageHeader,
   TechDocsReaderPageSubheader,
   TechDocsReaderPageContent,
 } from '@backstage/plugin-techdocs';
-import React from 'react';
 
 const DefaultTechDocsPage = () => {
   return (
-    <TechDocsReaderPage>
+    <Page themeId="documentation">
       <TechDocsReaderPageHeader />
       <TechDocsReaderPageSubheader />
       <TechDocsReaderPageContent />
-    </TechDocsReaderPage>
+    </Page>
   );
 };
```

Comment:
hmm maybe the default page can be just [`TechDocsReaderLayout`](https://github.com/backstage/backstage/blob/0df260a54fbb8c193dbedda7379a60a5dab0ff85/plugins/techdocs/src/reader/components/TechDocsReaderPage/TechDocsReaderPage.tsx#L60) instead then?

Reply:
Yeah, it seems kinda redundant to have this here at the moment (because it's exactly what gets rendered if you don't provide a custom reader page at all).

But if you _were_ to customize it, just having `<TechDocsReaderLayout />` wouldn't give a very good idea of where to start or what you could customize.

I found having this useful for playing around with the different possible customization mechanisms while fixing these bugs.  ...I imagine this will evolve a little more as we add a couple of addons to play around with, which will eventually inform what we ship in `create-app` too.

Reply:
good points, let's keep it!